db/db_impl/db_impl_secondary.h

```diff
@@ -269,6 +287,12 @@ class DBImplSecondary : public DBImpl {
     return s;
   }
 
+  bool OwnTablesAndLogs() const override {
+    InstrumentedMutexLock lock_guard(&mutex_);
+    return versions_->GetColumnFamilySet()->get_table_cache()->GetCapacity() !=
+           TableCache::kInfiniteCapacity;
```

Comment:
How about return `false` for now? If we need it later, we can add it back. Now it's now easy for people who read the code to understand.

Reply:
Sure, will do.